### PR TITLE
Astro 1370 integrate light theme styling

### PR DESCRIPTION
**Pull request by Full-lifey**
Adding for Deployment so design can review. 

Will take WIP tag off when ready for merge.

### CODE REVIEW

**Comment by Full-lifey** [comment]
Design has approved so we are ready for dev review.

File: .gitignore [hunk 1]
```diff
@@ -83,7 +83,7 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
+# dist
```
Comment (Full-lifey) [inline .gitignore:86]: So the production CSS can be served.

File: dark/package.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "description": "Astro Material Dark theme",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack serve --mode development",
-    "build": "webpack"
+    "start": "webpack serve --mode development --port 8081",
```
Comment (Full-lifey) [inline dark/package.json:7]: Added so dark can run in parallel with light

**Review by Full-lifey — COMMENTED** [review]
Most files can be ignored as they are immaterial for the CSS output. I added a lot of comments so the files are easier to parse.

A couple files and what they do that are important for QA but not for the theme build itself:
index.html - Forms the template the page is built on for viewing/testing components
my-layout.scss - Styling for index.html

The key files for the build of the theme are:
my-theme.scss - Colors, typography and shape
my-components.scss - Styling for each individual component

**Review by cskerbo — APPROVED** [review]
🥳

**Pull request merged**